packages/react-instantsearch/src/components/__snapshots__/Breadcrumb.test.js.snap

```diff
@@ -284,7 +306,9 @@ exports[`Breadcrumb outputs the default breadcrumb with no refinement 1`] = `
     >
       <span
         className="ais-Breadcrumb-separator"
-      />
+      >
+        &gt;
```

Comment:
is this change on purpose? I don't see the change in the test

Reply:
Yes since I move the `defaultProps` from the connector to the component it print now the default separator in the snapshot.

Reply:
Ah okay makes sense